Docs/source/Software/Tasks.rst

```diff
@@ -42,3 +42,59 @@ Process:
     2) If all of these checks are safe, the task will send the All Clear message and the Contactor On message across the CAN line.
     
     3) The task will then destroy itself since it is no longer needed
+
+Battery Balancing Task: Sugam Arora
+===================================
+
+Purpose
+    The solar car is powered by lithium ion cells. If these cells are overcharged, they experience accelerated degradation. This task has been 
+    implemented in an effort to maintain peak performance and health for the car's battery pack.
+
+Functionality
+    This task sets any battery module with a voltage that is higher than the minimum voltage of all the modules + a charging tolerance 
+    (subject to change) to discharge. Any modules that have a voltage that is equal to or less than the minimum voltage of all the 
+    modules in the system will have their discharge bit cleared. When clearing discharge bits, this task will access the MinionsASIC 
+    mutex. This task also uses the ``WDog_Mutex`` and toggles the ``WD_BALANCING`` bit of the ``WDog_BitMap``.
```

Comment:
I don't think you need to know about the discharge bits on the task level. I think it would be sufficient to just say that it discharges or charges cells instead of talking about how it does that on the register level. You also don't need to say which mutexes you are using since you do that under "Shared Resources"